README.md

```diff
@@ -5,14 +5,105 @@ This is the browser extension for Gitpod. It supports Chrome (see [Chrome Web St
 
  ![Gitpodify](./docs/github-injected.png "Gitpodify")
 
-## Build
 
-### Chrome & Firefox
+## Contributing
+
+Contributions to this project are welcome!
+
+### Issues
+
+We are currently tracking all issues related to the browser extension in the [`gitpod-io/gitpod`](https://github.com/gitpod-io/gitpod) repository.
+You can use the [`component: browser-extension`](https://github.com/gitpod-io/gitpod/issues?q=is%3Aissue+is%3Aopen+extension+label%3A%22component%3A+browser-extension%22) label to search for relevant issues including feature proposals and bug reports.
+
+### Development
+
+To make changes and test them, the easiest way is using Gitpod itself. The `.gitpod.yml` configuration for this project installs Google Chrome and runs a VNC server to allow users to see the Desktop of the Docker container. When a workspace starts, Google Chrome is started with the Gitpod extension automatically built from source and pre-installed, therefore you can preview your changes in the ephemeral Chrome browser. 🤯
```

Comment:
**thought**: Mind blowing indeed. Thanks @felladrin for adding this in https://github.com/gitpod-io/browser-extension/pull/55.